### PR TITLE
fix: documentation website for forma-v4

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

Fixes a broken build for the documentation website

### CODE REVIEW

File: packages/components/badge/src/Badge.mdx [hunk 1]
```diff
@@ -177,30 +174,14 @@ Tags are used to label or highlight items in the interface.
...
+        <Badge variant="positive">published</Badge>
       </div>
     </Flex>
   </div>
 </>
 ```
 
-- **Muted** - this `tagType` is deprecated, please use `tagType` `secondary` instead
-
-### Tag sizes
```
Comment (burakukula) [inline packages/components/badge/src/Badge.mdx:189]: @suevalov sorry if I didn't notice in the PR - size property was removed from Badge?

File: packages/forma-36-website/package.json [hunk 2]
```diff
@@ -44,6 +44,6 @@
   "scripts": {
     "build:website": "gatsby build",
     "develop": "gatsby develop",
```
Comment (burakukula) [inline packages/forma-36-website/package.json:46]: Maybe we would then just remove `develop` command and keep `start`? WDYT?

**Review by burakukula — APPROVED** [review]
@suevalov thx a lot for fixing this! 🙌

File: packages/forma-36-website/package.json (hunk 2 above)
Comment (suevalov) [reply packages/forma-36-website/package.json:46]: yep, will update this

File: packages/components/badge/src/Badge.mdx (hunk 1 above)
Comment (suevalov) [reply packages/components/badge/src/Badge.mdx:189]: yep, it was. shouldn't it be?

Comment (burakukula) [reply packages/components/badge/src/Badge.mdx:189]: yeah, I think it should - it was just introduced a couple of weeks ago.

Comment (suevalov) [reply packages/components/badge/src/Badge.mdx:189]: ok, I see. I'm going to fix it then

Comment (burakukula) [reply packages/components/badge/src/Badge.mdx:189]: Thank you! 😊

**Review by mshaaban0 — APPROVED** [review]

**Review by burakukula — APPROVED** [review]

**Review by nikazawila — APPROVED** [review]

**Pull request merged**